tests/run-pass/panic/std-panic-locations.rs

```diff
@@ -1,3 +1,5 @@
+// revisions: static dynamic
+//[static] no-prefer-dynamic
```

Comment:
(Adding this to ensure that the workaround for "crate ... required to be available in rlib format, but was not found in this form" errors works for `no-prefer-dynamic`, which seems on a different code path: https://github.com/rust-lang/rust/blob/e7ed7e835494219eeab314ace8beef40b695ce30/compiler/rustc_metadata/src/dependency_format.rs#L97-L99.)

Reply:
(This has been removed because I have added a [new test](https://github.com/rust-lang/miri/pull/1776/commits/38a5bc31b7c121dc769372c3f913e00946787fdf#diff-082b649bd007aa33c844d10756a15976c4d3639a2619f4e036299bd089b00e7c) for that purpose.)